### PR TITLE
This Month 2021-09: multiboot2

### PR DESCRIPTION
**Pull request by phip1611**
Hey! This is my first time adding information to the "This Month"-newsletter. I hope I didn't miss anything.

### CODE REVIEW

File: content/this-month/2021-09/index.md [hunk 1]
```diff
@@ -36,6 +36,29 @@ The `x86_64` crate provides various abstractions for `x86_64` systems, including
...
+- [prepared cargo workspace, as discussed in PR #79](https://github.com/rust-osdev/multiboot2/pull/86)
+
+The changes were published as <span class="gray">`v0.12.2`</span>.
+
+
+### [`multboot2-header`](https://github.com/rust-osdev/multiboot2) (**new**)
+The `multiboot2-header` crate provides abstraction types for the Multiboot2 header 
+and a builder struct to construct these headers. The corresponding repository was 
+prepared (<https://github.com/rust-osdev/multiboot2/pull/86>) and the initial release 
+is expected in early October. See <https://github.com/rust-osdev/multiboot2/pull/95> for more details.
```
Comment (phil-opp) [inline content/this-month/2021-09/index.md:60]:
```suggestion
prepared ([#86](https://github.com/rust-osdev/multiboot2/pull/86)) and the initial release 
is expected in early October. See [#95](https://github.com/rust-osdev/multiboot2/pull/95) for more details.
```

**Review by phil-opp — COMMENTED** [review]
Thanks a lot! I left a small formatting suggestion inline, otherwise this looks good!

Not directly related, but if you're planning to release the new header crate under the name `multiboot2-header`, I would recommend to squat that name on crates.io already. You can just push an empty dummy crate with a link to the repo. Otherwise, there is the risk that someone else squats that name when we mention it in the newsletter.

**Comment by phip1611** [comment]
> Not directly related, but if you're planning to release the new header crate under the name multiboot2-header, I would recommend to squat that name on crates.io already

Already done and marked rust-osdev team as co-owner.

**Comment by phil-opp** [comment]
Great, thanks!

**Pull request merged**